src/services/blockchain/__tests__/eth-bc-service.test.ts

```diff
@@ -55,6 +56,15 @@ describe('ETH service',  () => {
     const cid = new CID('bafyreic5p7grucmzx363ayxgoywb6d4qf5zjxgbqjixpkokbf5jtmdj5ni');
     const tx = await ethBc.sendTransaction(cid);
     expect(tx).toBeDefined();
+
+    // checking the timestamp against the snapshot is too brittle since if the test runs slowly it
+    // can be off slightly.  So we test it manually here instead.
+    const blockTimestamp = tx.blockTimestamp
+    delete tx.blockTimestamp
+    const startTimeSeconds = blockchainStartTime.getTime() / 1000
```

Comment:
```suggestion
    const startTimeSeconds = Math.floor(blockchainStartTime.getTime() / 1000)
```
???

Reply:
not necessary in this case since the start time is set by the test to an even multiple of 1000, but also doesn't hurt